### PR TITLE
Edit profile - Change password - Create new profile

### PR DESCRIPTION
**Pull request by duongtvph04550**

### CODE REVIEW

**Comment by DuyNguyen05** [comment]
:ok: :

**Comment by DuyNguyen05** [comment]
:ok:

**Comment by nguyenthanhluan92** [comment]
OK!

File: config/routes.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@
   delete "/logout", to: "sessions#destroy"
   get "/signup", to: "users#new"
   get "/qa", to: "questions#index"
-
+  resources :profiles, only: [:new, :edit, :create, :update]
```
Comment (n-luan) [inline config/routes.rb]: resource :profile, only [:edit. :update]

**Pull request merged**